solutions_builder/template_root/terraform/stages/1-bootstrap/terraform.tfvars

```diff
@@ -1,2 +1,18 @@
-project_id          = "{{project_id}}"
-storage_multiregion = "US"
+project_id             = "{{project_id}}"
+storage_multiregion    = "US"
+region                 = "{{gcp_region}}"
+zone                   = "{{gcp_zone}}"
```

Comment:
If you need the zone here for bastion, please add a question var in copier.yaml. You can name it as bastion_host_zone